lib/mayaUsd/render/vp2RenderDelegate/draw_item.cpp

```diff
@@ -25,21 +25,20 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 Data holder for its corresponding render item to facilitate parallelized evaluation.
 */
-HdVP2DrawItem::HdVP2DrawItem(HdVP2RenderDelegate* delegate, const HdRprimSharedData* sharedData)
+HdVP2DrawItem::HdVP2DrawItem(
+    HdVP2RenderDelegate*     delegate,
+    const HdRprimSharedData* sharedData)
     : HdDrawItem(sharedData)
     , _delegate(delegate)
 {
     // In the case of instancing, the ID of a proto has an attribute at the end,
-    // we keep this info in _renderItemName so if needed we can extract proto ID
+    // we keep this info in _drawItemName so if needed we can extract proto ID
     // and use it to figure out Rprim path for each instance. For example:
     //
     //   "/Proxy/TreePatch/Tree_1.proto_leaves_id0"
     //
-    _renderItemName = GetRprimID().GetText();
-    _renderItemName += TfStringPrintf("/DrawItem_%p", this).c_str();
-
-    _renderItemData._indexBuffer.reset(
-        new MHWRender::MIndexBuffer(MHWRender::MGeometry::kUnsignedInt32));
```

Comment:
Each render item has it's own index buffer, so the index buffer is not store on the draw item anymore.